assets/js/editor-components/search-list-control/search-list-control.tsx

```diff
@@ -123,7 +131,43 @@ const SelectedListItems = ( {
 			{ selectedCount > 0 ? (
 				<ul>
 					{ selected.map( ( item, i ) => (
-						<li key={ i }>
+						<li
+							key={ i }
```

Comment:
We should probably use a better `key`, now. I think `item` would work:
```suggestion
							key={ item }
```

This helps keeping the focus on the correct element when moving them.